api/desecapi/migrations/0020_user_email_verified.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.0.1 on 2022-01-14 13:39
+
+import datetime
+
+from django.db import migrations, models
+from django.db.models import F, Q
+
+
+def forwards_func(apps, schema_editor):
+    User = apps.get_model("desecapi", "User")
+    db_alias = schema_editor.connection.alias
+    filter = Q(is_active=True) | ~Q(last_login__isnull=True)
```

Comment:
Is this the same as `Q(is_active=True) | Q(last_login__isnull=False)`?

Reply:
yes, fixed